### ISSUE TITLE
navigation menu button doesn't alert screen readers when toggled

### ISSUE THREAD

**Issue by sharafeddeen** [StatisticsHelper/statistics-helper#11]
The navigation menu button works well for sighted users. Clicking the button implies that a state has changed (to open or close the nav menu). The screen reader can recognize it’s a navigational element and it’s a menu button, but it cannot know that the button’s state changes when the user clicks it. 
* You need to make it so the screen reader can read the elements inside the menu as well. Use the attribute `aria-expanded` to signal that to screen readers. 
* It’s recommended to set focus on the current link that’s being read by the screen reader.

**Comment by sharafeddeen** [comment]
I added the aria-expanded attribute and set automatic focus on the first link.
